contributors/onboarding/things-everyone-must-know.md

```diff
@@ -3,3 +3,7 @@
 1. **Always use the term "country/region" and never "country"** in discussions, documentation and software design. Mattermost, Inc. may work with public sector organizations with a high degree of sensitivity to regions where the label "country" may be controversial.  
 2. **Never sign on behalf of Mattermost, Inc.** Only the Mattermost, Inc. CEO can sign documents on behalf of the company, with the one and only exception being signing an on-site NDA \(non-disclosure agreement\) in order to enter a building. See [Risk Management in Agreements](../../operations/finance/risk-management/) documentation for instructions on having any agreement properly reviewed and initialed by someone Director-level or higher prior to CEO signature. 
 3. **Escalate sensitive topics** seen in public sites such as the forums to the [Community Escalations channel](https://community.mattermost.com/private-core/channels/community-escalations), and follow the [Community Response Policy](../../operations/operations/company-policies/community-response-policy).
+4. **Outside work** As an open-source company, Mattermost encourages Contributors to work on projects that are unrelated to Mattermost's business. Contributors do not require prior approval from Mattermost to work on projects that are unrelated to Mattermost's business or that do not pose a conflict of interest. Contributors should reference Mattermost's PIAA or contact hr@mattermost.com with questions. 
+FT Staff may work on outside projects (paid or unpaid) that are not related to Mattermost's business, do not represent a conflict of interest, and do not hinder their ability to fulfill their contractual obligations to Mattermost. Prior to commencing work on outside projects, staff must confirm no conflict of interest exists by obtaining written approval from their manager and MLT. 
```

Comment:
Unsure if we want to write out "FT" and "MLT" instead of use abbreviations.

Reply:
Updated "FT" but we use MLT as-is in the docs so 3/5 on it remaining as such. What do you think?